Client/src/components/layout/Header/Header.modules.css

```diff
@@ -0,0 +1,16 @@
+.Header {
+  height: 80px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: rgb(223, 223, 223);
+  padding: 0 40px;
+}
+
+.dark > .Header {
```

Comment:
TRy to always avoid markup structure oriented styles. It may change in a second.